packages/manager/apps/telecom/src/app/app.less

```diff
@@ -103,7 +103,7 @@
 @import './common/less/navbar.less';
 @import './common/less/variables.less';
 @import './common/less/mixins.less';
-@import './telecom/pack/migration/telecom-pack-migration.less';
+@import './telecom/pack/migration/pack-migration.less';
```

Comment:
Import could be moved to `packages/manager/apps/telecom/src/app/telecom/pack/migration/index.js` or does-it break everything (need more refactoring) ?

Reply:
If you want